app/models/miq_queue.rb

```diff
@@ -486,6 +486,14 @@ def deliver(requester = nil, &block)
     return status, message, result
   end
 
+  # @return true if completed, false if timed out
```

Comment:
This comment is not accurate anymore now that you are returning the status itself.